sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/join/CoGroupByKey.java

```diff
@@ -84,8 +95,19 @@
     return new CoGroupByKey<>();
   }
 
+  /**
+   * Add a {@link FilterOption#INNER_JOIN} to output {@code PCollection} of {@code CoGroupByKey}.
```

Comment:
I'd like to improve this Javadoc to include full description and specification here.